docs/getting_started_jax.ipynb

```diff
@@ -0,0 +1,669 @@
+        "\n",
+        "  print(jax.jit(shmaped_f).lower(x).as_text())\n",
+        "\n",
+        "print(run_shard_map())"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yug6aSqXc_Sv"
+      },
+      "source": [
+        "With the custom calls and `HloShardingV2`s, it's getting pretty confusing. Let's look at what Shardy gives:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 10,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "1hCGbWnbdHa-",
+        "outputId": "ee16708c-eb29-44dd-c807-031a3728a6c2"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "module @jit_body attributes {mhlo.num_partitions = 8 : i32, mhlo.num_replicas = 1 : i32} {\n",
+            "  sdy.mesh @mesh = <[\"data\"=4, \"model\"=2]>\n",
+            "  func.func public @main(%arg0: tensor<32x64xf32>) -> (tensor<32x64xf32> {jax.result_info = \"\"}) {\n",
+            "    %0 = sdy.manual_computation(%arg0) in_shardings=[<@mesh, [{\"data\"}, {}]>] out_shardings=[<@mesh, [{}, {}]>] manual_axes={\"data\"} (%arg1: tensor<8x64xf32>) {\n",
+            "      %1 = sdy.sharding_constraint %arg1 <@mesh, [{\"model\"}, {?}]> : tensor<8x64xf32>\n",
+            "      %2 = \"stablehlo.all_gather\"(%1) <{all_gather_dim = 0 : i64, channel_handle = #stablehlo.channel_handle<handle = 1, type = 1>, replica_groups = dense<[[0, 2, 4, 6], [1, 3, 5, 7]]> : tensor<2x4xi64>, use_global_device_ids}> : (tensor<8x64xf32>) -> tensor<32x64xf32>\n",
+            "      sdy.return %2 : tensor<32x64xf32>\n",
+            "    } : (tensor<32x64xf32>) -> tensor<32x64xf32>\n",
+            "    return %0 : tensor<32x64xf32>\n",
+            "  }\n",
+            "}\n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "jax.config.update(\"jax_use_shardy_partitioner\", True)\n",
+        "run_shard_map()\n",
+        "jax.config.update(\"jax_use_shardy_partitioner\", False)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f55laSGGdMP2"
+      },
+      "source": [
+        "We now:\n",
+        "\n",
+        "- Have a single op called `sdy.manual_computation` which holds:\n",
+        "  - the `in_specs`\n",
+        "  - the `out_specs`\n",
+        "  - the body of the shard_map\n",
+        "  - the inverse of the `auto` axes which we call `manual_axes`\n",
+        "\n",
+        "A lot easier to read!\n",
+        "\n",
+        "Note that `manual_axes` is always equal to the axes in the `mesh` but not in the `auto = frozenset([...])`.\n",
+        "\n",
+        "Also, this will be relaxed in the JAX side soon, but you will be able to use `UNSPECIFIED` on in/out specs and have `auto` axes propagate from inside to outside the shard_map body (and vice-versa)."
```

Comment:
Since we don't want to talk about `auto` and `UNSPECIFIED` just yet lets remove these two lines as well